### PR TITLE
Add the UDOO Bolt to Blinka

### PR DESCRIPTION
**Pull request by geekguy-wy**
I am having a difficult time getting the ryzen_v1605b/pin.py and board file udoo_bolt_v8.pyset up correctly. I have tried setting everything up the way others are set up, but nothing has worked so far.

### CODE REVIEW

File: src/microcontroller/__init__.py [hunk 1]
```diff
@@ -74,7 +74,9 @@ def __repr__(self):
     from adafruit_blinka.microcontroller.nxp_lpc4330 import *
 elif chip_id == ap_chip.MIPS24KC:
     from adafruit_blinka.microcontroller.atheros.ar9331.pin import *
-elif chip_id == ap_chip.MIPS24KEC:
-    from adafruit_blinka.microcontroller.mips24kec.pin import *
```
Comment (makermelissa) [inline src/microcontroller/__init__.py:78]: Please add the MIPS24KEC code section back in. Otherwise it will break Onion Omega 2 support.

Comment (geekguy-wy) [reply src/microcontroller/__init__.py:78]: I did not remove any code, at least not on purpose, but I know what happened. That code was (is) right above where I added the ryzen code. I cloned off the MIPS24KEC code to add the ryzen stuff and forgot to put the original code back. All restored!

Comment (makermelissa) [reply src/microcontroller/__init__.py:78]: Thanks!

**Comment by geekguy-wy** [comment]
Due to problems I am having with adding the UDOO Bolt to Blinka, I am closing this issue for now. I will re-open it when I am able to make more progress.

**Pull request closed without merging**